pkg/kubelet/dockertools/exec.go

```diff
@@ -121,6 +126,14 @@ func (*NativeExecHandler) ExecInContainer(client DockerInterface, container *doc
 	if err != nil {
 		return err
 	}
+	if timeout > 0 {
+		defer time.AfterFunc(timeout, func() {
+			// FIXME: we should kill the process in the container,
+			// but I couldn't find anything in the Docker API docs
+			// on how to do it with the *Exec APIs.
```

Comment:
StartExec is blocking. It didn't used to be. This code probably wasn't updated when we switched it to blocking.